parcel_test.go

```diff
@@ -31,57 +32,98 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	assert.NoError(t, err)
```

Comment:
Тут точно нужен require, так как если ошибка БД, то тест нужно завершить
```suggestion
	require.NoError(t, err)
```